vnext/Microsoft.ReactNative/ReactRootView.cpp

```diff
@@ -283,6 +303,65 @@ void ReactRootView::EnsureLoadingUI() noexcept {
   }
 }
 
+void ReactRootView::HideDebuggerPausedOverlay() noexcept {
+  m_isDebuggerPausedOverlayOpen = false;
+  if (m_debuggerPausedFlyout) {
+    m_debuggerPausedFlyout.Hide();
+    m_debuggerPausedFlyout = nullptr;
+  }
+}
+
+void ReactRootView::ShowDebuggerPausedOverlay(
+    const std::string &message,
+    const std::function<void()> &onResume) noexcept {
+  // Initialize content
+  const xaml::Controls::Grid contentGrid;
```

Comment:
This could probably be converted to a xaml string or something, or a xaml component, but since we don't have a XAML compiler outside visual studio, I prefer this to be imperative WinUI set up :|